oxidation/client/src/views/HomePage.vue

```diff
@@ -350,187 +333,261 @@ function onForgottenPasswordClick(): void {
   console.log('forgotten password!');
 }
 
-async function openJoinByLinkModal(): Promise<void> {
-  const modal = await modalController.create({
-    component: JoinByLinkModal,
-    cssClass: 'join-by-link-modal'
-  });
-  await modal.present();
-
-  const { data, role } = await modal.onWillDismiss();
-
-  if (role === 'confirm') {
-    console.log(data);
-  }
-}
-
-async function openCreateOrganizationModal(): Promise<void> {
-  const modal = await modalController.create({
-    component: CreateOrganization,
-    canDismiss: canDismissModal,
-    cssClass: 'create-organization-modal'
+async function openPopover(ev: Event): Promise<void> {
+  const popover = await popoverController.create({
+    component: HomePagePopover,
+    cssClass: 'homepage-popover',
+    event: ev,
+    showBackdrop: false
   });
-  await modal.present();
-
-  const { data, role } = await modal.onWillDismiss();
-
-  if (role === 'confirm') {
-    console.log(data);
-  }
+  await popover.present();
 }
 
-async function canDismissModal(): Promise<boolean> {
-  const alert = await createAlert(
-    t('AlertConfirmation.areYouSure'),
-    t('AlertConfirmation.infoNotSaved'),
-    t('AlertConfirmation.cancel'),
-    t('AlertConfirmation.ok')
-  );
-
-  await alert.present();
-
-  const { role } = await alert.onDidDismiss();
-  return role === 'confirm';
-}
 </script>
 
 <style lang="scss" scoped>
 #page {
   height: 100vh;
+  background: var(--parsec-color-light-secondary-inversed-contrast);
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
   overflow: hidden;
   margin: 0 auto;
   align-items: center;
+  position: relative;
+  z-index: -4;
+}
 
-  #topbar {
+.sidebar {
+  display: flex;
+  height: 100vh;
+  width: 40vw;
+  padding: 2rem 0;
+  background: var(--parsec-color-light-gradient);
+  position: relative;
+  justify-content: flex-end;
+  z-index: -3;
+
+  &::before {
+    content: '';
+    position: absolute;
+    z-index: -2;
+    top: 0;
+    right: 0;
+    width: 100vw;
+    height: 100vh;
+    background: url('../assets/images/shapes-bg.svg') repeat center;
+    background-size: cover;
+  }
+
+  .sidebar-content {
     display: flex;
+    flex-direction: column;
     align-items: center;
-    flex-basis: 5em;
-    flex-grow: 0;
-    flex-shrink: 0;
-    background: var(--parsec-color-light-secondary-inverted-contrast);
-    width: 100vw;
-    justify-content: center;
-    margin-bottom: 5em;
+    margin-left: 2rem;
 
-    ion-col {
+    &__titles {
       display: flex;
-      align-items: center;
-    }
-
-    ion-col:first-child {
+      flex-direction: column;
       justify-content: center;
+      flex-grow: 2;
+      margin-right: 2rem;
+      max-width: 500px;
```

Comment:
on what is based this '500px'?

Reply:
Add the `max-width` variable to the global style file and replace `px` with this variable